blockly-core/core/utils/variables.js

```diff
@@ -42,13 +42,16 @@ Blockly.Variables.NAME_TYPE_LOCAL = 'LOCALVARIABLE';
 /**
  * Find all user-created variables.
  * Currently searches the main blockspace only
- * @param {Blockly.Block=} opt_block Optional root block.
+ * @param {Array.<Blockly.Block>} opt_blocks Optional root blocks.
```

Comment:
May want to retain the `=` or bracket like `[opt_blocks]` to mark as jsdoc optional